app/assets/javascripts/discourse/package.json

```diff
@@ -61,6 +61,7 @@
     "pretty-text": "1.0.0",
     "select-kit": "1.0.0",
     "tippy.js": "^6.3.7",
+    "truth-helpers": "1.0.0",
```

Comment:
Should this be in dev dependencies? Not sure what's the difference in the context of the discourse app

Reply:
There's no real technical difference - we always install devDependencies, even when doing a 'production' Ember build. Happy to take your advice on whatever would be 'normal' for ember apps.

Reply:
I think usually pretty much everything goes into devDeps. deps could include stuff like express.js and fastboot-related packages.

Reply:
Yeah, normally almost everything would be in devDeps, but I see that there were a bunch of things here in regular deps so I wasn't sure whether that was needed for some reason (i.e. whether `discourse` is being consumed as a package by something else).

Reply:
Discourse definitely isn't being consumed as a package anywhere, so I guess let's try moving all this stuff to devDeps 👍